lib/client.js

```diff
@@ -73,7 +73,12 @@ class Client {
   constructor(room, socket, rtoken) {
     const {remoteAddress, remotePort} = socket.request.connection;
     this.socket = socket;
-    this.ip = remoteAddress;
+    if (CONFIG.get("considerProxyForwardedForHeaders")) {
+      this.ip = socket.request.headers["x-forwarded-for"] || remoteAddress;
+      this.ip = this.ip.split(",")[0];
+    } else {
```

Comment:
>else on same line
kys

Reply:
The one-true-brace-style will someday win.  You will see.